src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsEnforcementSpec.groovy

```diff
@@ -473,4 +477,28 @@ class PriceFloorsEnforcementSpec extends PriceFloorsBaseSpec {
         assert response.seatbid?.first()?.bid?.collect { it.id } == [bidResponse.seatbid.first().bid.first().id]
         assert response.seatbid.first().bid.collect { it.price } == [floorValue]
     }
+
+    def "PBS shouldn't return error when price floor enforcement service enforce bid request"() {
+        given: "BidRequestWithFloors with Stored Action Response and Floor enabled = false"
+        def storedResponseId = PBSUtils.randomNumber
+        def bidRequest = getBidRequestWithFloors().tap {
+            ext.prebid.floors.enabled = false
+            imp[0].ext.prebid.storedAuctionResponse = new StoredAuctionResponse(id: storedResponseId)
+        }
+
+        and: "Stored response in DB"
+        def storedAuctionResponse = SeatBid.getStoredResponse(bidRequest)
+        def storedResponse = new StoredResponse(resid: storedResponseId, storedAuctionResponse: storedAuctionResponse)
+        storedResponseDao.save(storedResponse)
+
+        and: "Account with enabled fetch, fetch.url in the DB"
+        def account = getAccountWithEnabledFetch(bidRequest.site.publisher.id)
+        accountDao.save(account)
+
+        when: "PBS processes auction request"
+        floorsPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Request shouldn't fail with an error"
+        noExceptionThrown()
```

Comment:
Please validate with an assertion against the response itself too.

Reply:
Again, please do a validation that there is an actial response and there are no errors.